### ISSUE TITLE
[Enhancement] Convert calendar to Ionic datepicker

### ISSUE THREAD

**Issue by mark-drastrup** [kirbydesign/designsystem#2461]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
Convert the current Kirby calendar to [Ionic datepicker](https://ionicframework.com/docs/api/datetime).

## Describe the solution you'd like
The Ionic datepicker largely contains the same functionality as the Kirby calendar, but exposes a different API (primarily for enabling/disabling dates). A possible solution could be to make a wrapper component for the Ionic datepicker, that takes the same inputs as the Kirby calendar and then converts it to the format required by the Ionic datepicker.

## Are there any additional context?
Currently there is no option to configure the header of the Ionic datepicker.
<img width="386" alt="Screenshot 2022-09-06 at 12 47 38" src="https://user-images.githubusercontent.com/20282214/188617011-7596b475-9c82-4332-bf70-c708b2f9f09e.png">

Additionally, when clicking the month the datepicker displays a whee. This is not configurable.
<img width="374" alt="Screenshot 2022-09-06 at 12 49 27" src="https://user-images.githubusercontent.com/20282214/188617847-4ee53a21-90ac-41e3-80f0-91f50899b3db.png">

These two issues prevents the possibility of implementing the UI 100% the same as the current calendar. As a result this issue needs to be UX refined.

<hr />

### Refinement

- [x] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [x] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

**Comment by RasmusKjeldgaard** [comment]
No longer relevant.
